### ISSUE TITLE
LoadQueue.remove() does not remove loaded items

### ISSUE THREAD

**Issue by michaelwoods** [CreateJS/PreloadJS#123]
Line 4782 of preloadjs-0.6.0.combined.js uses getResult() which returns an item from the _loadResults (or _loadRawResults).

```js
var r = this.getResult(item);
```
This object does not contain an id or src property, therefore after the check for 'r' on line 4802, the incorrect object is disposed of, unsuccessfully. The two deletes above it are also extraneous as those occur in the _disposeItem().
```js
if (r) {
    delete this._loadItemsById[r.id];
    delete this._loadItemsBySrc[r.src];
    this._disposeItem(r);
} else {
...
```
Changing the first line to use getItem() solves this issue. The two delete lines can also be removed.
```js
var r = this.getItem(item);
...
if (r) {
    this._disposeItem(r);
} else {
...
```

**Comment by azack** [comment]
The extra deletes appear to have been removed previously by 4741aaea4ad5bad66a12ee67f803d2ee64e18cbe

**Comment by azack** [comment]
Oops, just realized that that commit also called getItem.

**Comment by lannymcnie** [comment]
@azack Your comments aren't clear. What is the status of this bug?

**Comment by azack** [comment]
@lannymcnie It appears fixed indirectly by 4741aae, so this can be closed.
